src/doc/cli_manual/functions.rst

```diff
@@ -99,7 +96,7 @@ AddMachineProfile
   AddMachineProfile(MachineProfile) -> integer
 
 
-MachineProfile : type
+MachineProfile : STARTING_VALUE
```

Comment:
Is this type of `STARTING_VALUE` really correct? It feels wrong.

Reply:
This, along with the word AMBIGUOUS are meant to serve as flags for where we will need to do the update by hand. In this case, a type keyword (such as string, double, etc.) could not be found, so it places STARTING_VALUE there.

Reply:
Ok, that makes sense.